test_tools/fio/fio.py

```diff
@@ -25,7 +26,7 @@ def __init__(self, executor_obj=None):
         self.base_cmd_parameters: FioParam = None
         self.global_cmd_parameters: FioParam = None
 
-    def create_command(self, output_type=FioOutput.json):
+    def create_command(self, output_type=FioOutput.json, alloc_size: Size = None):
```

Comment:
I think both alloc_size and output_type should be elsewhere - this was by design a simple method with all the default values, everything configurable should be set in separate functions

Reply:
actually json should probably be set in this method, just without passing it as an argument